dotnet/src/Planners/Planners.Core/Handlebars/CreatePlanPrompt.handlebars

```diff
@@ -127,7 +182,7 @@ IMPORTANT: You can only use the helpers that are listed above. Do not use any ot
 - Do not make up values. Use the helpers to generate the data you need or extract it from the goal.
 - Keep data well-defined. Each variable should have a unique name. Create and assign each variable only once.
 - Be extremely careful about types. For example, if you pass an array to a helper that expects a number, the template will error out.
-- Avoid using loops. Try a solution without loops before you deploy a loop.
+{{#if allowLoops}}- Avoid using loops. Try a solution without before you deploy a loop.{{/if}}
 - There is no need to check your results in the template.
 - Do not nest sub-expressions or helpers because it will cause the template to error out.
 - Each step should contain only one helper call.
```

Comment:
We definitely need to figure out how to make this more maintainable

Reply:
Yeah, I'm thinking we can split it with partials, and it'll be more digestible.